src/main/java/org/breedinginsight/services/validators/TraitValidatorError.java

```diff
@@ -155,7 +155,7 @@ public ValidationError getCharLimitTraitAttributeMsg() {
 
     @Override
     public ValidationError getCharLimitMethodDescriptionMsg() {
-        return new ValidationError("method.description", "Method description exceeds 30 character limit", HttpStatus.UNPROCESSABLE_ENTITY);
+        return new ValidationError("Method Description", "Method description exceeds 30 character limit", HttpStatus.UNPROCESSABLE_ENTITY);
     }
 
     @Override
```

Comment:
Should the terminology here and in TraitValidationError be changed to name rather than trait name? Should it also be capitalized?

Reply:
Trait name is now a concatenation of entity + attribute; whereas, obs var. name is the Name. So my feeling is to keep trait name as is for now, but I updated the rest of the field names to have consistent formatting/capitalization.